active/CVE-2018-11214

```diff
@@ -40,7 +40,7 @@ devel_libjpeg6b: needed
 
 Patches_libjpeg-turbo:
  upstream: https://github.com/libjpeg-turbo/libjpeg-turbo/commit/6709e4a0cfa44d4f54ee8ad05753d4aa9260cb91
-upstream_libjpeg-turbo: needs-triage
+upstream_libjpeg-turbo: released (1.3.0-0ubuntu2.1)
```

Comment:
When you see `ubuntux.x` in the version, that means there are ubuntu-specific changes in that version, i.e. it's not an upstream version. [According to Debian](https://security-tracker.debian.org/tracker/CVE-2018-11214), this is fixed in 1:1.3.1-12+deb8u1 and 1:1.4.2-1.